GVFS/GVFS.Platform.Mac/MacPlatform.cs

```diff
@@ -16,7 +17,7 @@ public MacPlatform()
                 executableExtension: string.Empty,
                 installerExtension: ".dmg",
                 underConstruction: new UnderConstructionFlags(
-                    supportsGVFSService: false,
+                    supportsGVFSService: true,

```

Comment:
We've been removing flags from UnderContstructionFlags when they become true on all platforms that we currently have rather than setting them to true.

Reply:
Removed all occurrences.